src/main/java/com/hillel/piatnytsia/homeworks/homework7/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,37 @@
+package com.hillel.piatnytsia.homeworks.homework7.coffee.order;
+
+import java.util.*;
+
+
+public class CoffeeOrderBoard {
+
+    Queue<Order> queue = new LinkedList<>() {
+    };
```

Comment:
why do you need the brackets at linkkedLidst  {}?

Reply:
When I was changing the structure, it came out by accident, I corrected it